Update Main.ahk

sieucapoccho3717 opened the pull request:
Dino Update

sieucapoccho3717 committed:
Update Main.ahk

Krxnkos commented:
Just tested this, it works well. Just wish there was integration with the centre of the maps events

sieucapoccho3717 committed:
Update README.md

sieucapoccho3717 committed:
Update README.md

sieucapoccho3717 committed:
Update README.md

sieucapoccho3717 committed:
Update README.md

sieucapoccho3717 commented:
have 0 english skill so have to make multiple changes

henris157 commented:
viraeg can yu make a new dino one with the seeds that are in rn and i bought the premium macro so cacn you figure out a way to do something with the dino event

sieucapoccho3717 committed:
added corn seed

sieucapoccho3717 committed:
Update 12 July

Added Giant Pinecone Seed, fixed discord pings for valuable seed which I accidentally broke with the first dino update

NotchedFox commented:
Any progress being made on the new Egg shop?

sieucapoccho3717 committed:
Updated for Gear Shop and new Egg Shop

Updated medium toy and medium treat to gear shop list
Updated so the macro works for new Egg Shop which is now using UI like any other shop

sieucapoccho3717 commented:
@NotchedFox Can you help me test it out? Seed and Gear buying shouldn't have any problem but Im worried about the new egg shop since I had to make it from scratch. For any bug you found, please let me know so I can fix it as soon as I wake up.

sieucapoccho3717 committed:
Update README.md

NotchedFox commented:
> @NotchedFox Can you help me test it out? Seed and Gear buying shouldn't have any problem but Im worried about the new egg shop since I had to make it from scratch. For any bug you found, please let me know so I can fix it as soon as I wake up.

@sieucapoccho3717 Testing now, so far so good on first cycle. ill keep you updated if it gets buggy

sieucapoccho3717 committed:
Added Levelup Lollipop

I didn't saw it at the end of the shop

sieucapoccho3717 committed:
Merge branch 'DinoUpdate' of https://github.com/sieucapoccho3717/Virage-Grow-A-Garden-Macro into DinoUpdate

sieucapoccho3717 commented:
@NotchedFox , can you use the newer version I just pushed? I forgot to add leveling lollipop to the gear shop.

sieucapoccho3717 commented:
There may be a bug with the Discord webhook that sends an incorrect status of the gear shop sequence to your Discord bot, indicating that selected gear shop items were bought but reporting "Failed To Detect Seed Shop Opening [Error]" instead.

sieucapoccho3717 committed:
Bug fix

Discord webhook should works normally for Gear shop

sieucapoccho3717 committed:
made auto-align more consistent?

I'm not really sure what the cause for this was; sometimes when auto-aligning to the step of switching between camera modes, the macro setting opens but doesn't navigate to the next tab. Could only be triggered on the first auto-align sequence

Just made the wait time a tiny bit longer, but it can also just be msedgeWebView2 causing the problem.

sieucapoccho3717 committed:
Global speed change

Set "Stable" as the default speed, and temporarily remove all other speed settings since they may be the cause of some bugs, and because I don't really think they are needed.
Slowed down some actions that are too fast.

sieucapoccho3717 commented:
The inconsistency in changing camera mode still exists even though I've tried plenty of ways to fix it; maybe it's just a Roblox thing after all.

sieucapoccho3717 committed:
Update README.md

Krxnkos commented:
When running the latest version of this PR, it keeps bringing me to the seed stall as opposed to the seed stall. Auto-align doesn't work even when running as an administrator.

Specs used to test:
CPU: AMD EPYC™ 9845
RAM: 1.48TB DDR4 ECC Memory @ 3100MHz
GPU: 6x NVIDIA RTX 4090 24GB VRAM
PSU: 1x Main 10kW PSU | 1x Redundant 5kW PSU

Krxnkos commented:
Running on a Virtual machine at the moment since I'm away from home.

sieucapoccho3717 commented:
@Krxnkos could you send me a screen recording showing how the macro went wrong for you? Streamable or Discord is fine.

Krxnkos commented:
Sure, what's your discord?

sieucapoccho3717 commented:
@Krxnkos I put the link in the README file

Rayozin1337 commented:
Dude, i just saw a Glitch, it's that the Pet eggs instead of zooming until one before the First Person, just completely Zooms to the First Person Mode, think you should fix this.

rlane85 commented:
Thanks for keeping this going! I was updating the new items in the script, but came back to see if the new egg shop had been updated. Which is when I discovered your branch here. I've been testing it on a few different computers/accounts and I haven't noticed any glitches! Sometimes the script will get out-of-whack like it always did, but it still works about 95% of the time, including the new egg shop! So thanks again!

guamothy commented:
https://github.com/user-attachments/assets/59dad325-1c69-4c37-9764-5d712afd4263

this happens to me

guamothy commented:
i mightve missed something in the readme.md but yeah

Duck-tacular commented:
so the thing above this was fixed by me running as admin but it seemed to glitch when I go to the gear/egg/seed it starts 3 things down so like it would start on blueberry instead of carrot, this seemed to make the restock roblox icon show up after completing the gear shop cycle

Duck-tacular commented:
never mind it seemed to fix on everything besides the egg shop when i did all the instructions correct

xeq-lr commented:
For some reason the time is off by ~25 seconds. Not sure if it's just my system but running as admin had no effect.

sieucapoccho3717 committed:
Update README.md

thatashleyparks commented:
Is Virage planning to release any updates? I am not good with inserting those scripts, etc. so IDK where to start with that ..... hoping to an updated link to the actual macro?

8-del commented:
Auto reconnect doesn't do anything

bg031011 commented:
When will the Zen update link come out?
